### PR TITLE
docs: change relative URLs to absolute URLs to fix broken links

### PR DESCRIPTION
**Pull request by laljikanjareeya**
Fixes https://github.com/googleapis/nodejs-redis/issues/385

### CODE REVIEW

File: google/cloud/managedidentities/v1/resource.proto [hunk 1]
```diff
@@ -71,7 +71,7 @@ message Domain {
   map<string, string> labels = 2 [(google.api.field_behavior) = OPTIONAL];
 
   // Optional. The full names of the Google Compute Engine
-  // [networks](/compute/docs/networks-and-firewalls#networks) the domain
+  // [networks](https://cloud.google.com/compute/docs/networks-and-firewalls#networks) the domain
```
Comment (frankyn) [inline google/cloud/managedidentities/v1/resource.proto:74]: @danoscarmike do you know if we would need to make this change in the Redis protos instead?

Comment (danoscarmike) [reply google/cloud/managedidentities/v1/resource.proto:74]: Yep, we should do this upstream. This change will get blown away by the next sync.

Comment (JustinBeckwith) [reply google/cloud/managedidentities/v1/resource.proto:74]: @frankyn just checking in - anyone cut a CL here yet?

Comment (frankyn) [reply google/cloud/managedidentities/v1/resource.proto:74]: No one has cut a CL yet. Apologies, I missed these two last responses.

Comment (frankyn) [reply google/cloud/managedidentities/v1/resource.proto:74]: Looks like it got resolved in: https://github.com/googleapis/googleapis/commit/8be3770465a6cf19bea97a84b3e82b113b0239b6

**Pull request closed without merging**